lib/v2/datasource/remote/util/CacheKeys.dart

```diff
@@ -0,0 +1,6 @@
+class CacheKeys {
```

Comment:
i dont think this should live in the utils. 
also not in remote data source. This should be in the local data source

Reply:
Moved and renamed